client/state/data-layer/wpcom/read/sites/notification-subscriptions/new/index.js

```diff
@@ -0,0 +1,58 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { READER_SUBSCRIBE_TO_NEW_POST_NOTIFICATIONS } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+import { translate } from 'i18n-calypso';
+import { bypassDataLayer } from 'state/data-layer/utils';
+import { unsubscribeToNewPostNotifications } from 'state/reader/follows/actions';
+
+export function fromApi( response ) {
+	const isAdded = !! ( response && response.success );
+	if ( ! isAdded ) {
+		throw new Error(
+			`Subscription to new post notifications failed with response: ${ JSON.stringify( response ) }`
+		);
+	}
+
+	return response;
+}
+
+export function requestNotificationSubscription( action ) {
+	return http(
+		{
+			path: `/read/sites/${ action.payload.blogId }/notification-subscriptions/new`,
+			method: 'POST',
+			apiNamespace: 'wpcom/v2',
+		},
+		action
+	);
+}
+
+export function receiveNotificationSubscriptionError( action ) {
+	return [
+		errorNotice(
+			translate( 'Sorry, we had a problem subscribing you to notifications. Please try again.' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Sorry, we had a problem subscribing. Please try again.' )` *ES Score: 9*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/post-notifications-subscription-data-layer)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).